### DISCUSSION TITLE
Is it really the end of BrainVISA forum ?

### DISCUSSION THREAD

**Discussion by sapetnioc** [brainvisa/casa-distro#178] [Ideas]
I saw this new GithHub feature allowing to have discussion without creating issues. I am being testing it. It is a serious candidate to completely disable BrainVISA forum (put it read-only) for the next release.

**Comment by denisri** [comment]
It can be an option. Our forum is old and unmaintained.
Is it possible to have this "discussion" feature at the github organization level (brainvisa) rather than project level ? I think users will often not really know in which sub-project their question should go, so having a coarser level (or a general discussion forum) seems better to me. Or maybe using a project designed for it, or maybe brainvisa.github.io but this project is rather meant for developers whereas here we are oriented to users I think ?
Other question: if we switch to this system we will have an even tighter dependency on github and microsoft. Is it OK ?
